app/views/search/bento.html.erb

```diff
@@ -1,29 +1,46 @@
 <%= render partial: "form" %>
 
-<div class="col-sm-4">
-  <h2>Articles and Stuff</h2>
-  <div id="articles_content">
-    Loading results...
-    <i class="fa fa-spinner fa-spin"></i>
+<div class="row">
+  <div class="col-sm-4">
+    <h2>Articles and Stuff</h2>
+    <div id="articles_content">
+      Loading results...
+      <i class="fa fa-spinner fa-spin"></i>
+    </div>
   </div>
-</div>
 
-<div class="col-sm-4">
-  <h2>Books and Stuff</h2>
-  <div id="books_content">
-    Loading results...
-    <i class="fa fa-spinner fa-spin"></i>
+  <div class="col-sm-4">
+    <h2>Books and Stuff</h2>
+    <div id="books_content">
+      Loading results...
+      <i class="fa fa-spinner fa-spin"></i>
+    </div>
   </div>
-</div>
 
-<div class="col-sm-4">
-  <h2>Website and Guides</h2>
-  <div id="website_content">
-    Loading results...
-    <i class="fa fa-spinner fa-spin"></i>
+  <div class="col-sm-4">
+    <h2>Website and Guides</h2>
+    <div id="website_content">
+      Loading results...
+      <i class="fa fa-spinner fa-spin"></i>
+    </div>
   </div>
 </div>
 
+<div class="row">
```

Comment:
Totally minor at this stage, but can you add some visual separation between the results and this message?

An ```<hr />``` or something simple would be enough for now.